github_api/voting.py

```diff
@@ -160,21 +161,29 @@ def parse_review_for_vote(state):
 
 def parse_reaction_for_vote(body):
     """ turns a comment reaction into a vote, if possible """
-    try:
```

Comment:
It was important because reactions content can be `+1` or `-1`, but also a string value. Hence the try/except when casting the content to an `int`. This PR changes that to comparing the string value.